### PR TITLE
Use input type="color" instead of ColorPicker2

### LINKED ISSUE #829: Colorpicker is too clumsy and is very limited in terms of choice in Admin->Facilities->Add
Instead of javascript, HTML input type "color" be used as it is more customizable and looks beautiful

Existing

Proposed

### PR DESCRIPTION
**Pull request by Jdew192837**
Will follow up with a code cleanup commit in the morning (hopefully). However, functionality works.
- Also need to apply class="text" to span elements

UI Changes:
- Reorganized the color picker row, looks nicer now

Fixes #829

### CODE REVIEW

**Comment by Jdew192837** [comment]
Before (Add Facility):
==

screenshot 59

After (Add Facility):
==

screenshot 64

Before (Edit Facility):
==

screenshot 60

After (Edit Facility):
==

screenshot 66


PTAL @teryhill @aethelwulffe 

Thanks!

File: interface/usergroup/facilities_add.php [hunk 1]
```diff
@@ -42,272 +42,375 @@
...
-</table>
+            function displayAlert() {
+                if (document.getElementById('primary_business_entity').checked==false)
+                    alert("<?php echo addslashes(xl('Primary Business Entity tax id is used as account id for NewCrop ePrescription. Changing the facility will affect the working in NewCrop.'));?>");
+                else if (document.getElementById('primary_business_entity').checked==true)
+                    alert("<?php echo addslashes(xl('Once the Primary Business Facility is set, it should not be changed. Changing the facility will affect the working in NewCrop ePrescription.'));?>");
+            }
 
-<br>
+            function changeColor(id) {
```
Comment (Jdew192837) [inline interface/usergroup/facilities_add.php]: Note: Here's the function I added

File: interface/usergroup/facilities_add.php [hunk 2]
```diff
@@ -42,272 +42,375 @@
...
+                <tr>
+                    <td>
+                        <span class='text'><?php echo xlt('Service Location'); ?>: </span>
+                    </td>
+                    <td>
+                        <input type='checkbox' name='service_location' value = '1'>
+                    </td>
+                    <td>&nbsp;</td>
+                </tr>
+                <tr>
```
Comment (Jdew192837) [inline interface/usergroup/facilities_add.php]: Note: Here's where I made changes to the color picker

File: interface/usergroup/facility_admin.php [hunk 3]
```diff
@@ -28,221 +28,426 @@
...
+    }
 
-  function changeColor(id) {
-    // Gets the value from the color input
-    newColor = document.getElementById(id).value;
-    // Set the display input to the new color
-    document.getElementById('ncolor').value = newColor;
-  }
-</script>
+    function changeColor(id) {
```
Comment (Jdew192837) [inline interface/usergroup/facility_admin.php]: Note: Where I added the function

File: interface/usergroup/facility_admin.php [hunk 4]
```diff
@@ -28,221 +28,426 @@
...
-    </tr>
+        </tr>
+        <tr>
+          <td>
+            <span class="text"><?php echo htmlspecialchars(xl('Color'),ENT_QUOTES); ?>:</span>
+            <span class="mandatory">&nbsp;*</span>
+          </td>
+          <td>
+            <span class="text">Pick:&nbsp;
+              <input
```
Comment (Jdew192837) [inline interface/usergroup/facility_admin.php]: Note: Where I added my colorPicker input

File: interface/usergroup/facilities_add.php [hunk 5]
```diff
@@ -42,267 +42,375 @@
...
+                    <td>
+                        <span class="text"><?php echo htmlspecialchars(xl('Color'),ENT_QUOTES); ?>:</span>
+                        <span class="mandatory">&nbsp;*</span>
+                    </td>
+                    <td>
+                        <span class="text">Pick:&nbsp;<input type="color" id="colorPicker" onChange="changeColor('colorPicker')"></span>
```
Comment (Jdew192837) [inline interface/usergroup/facilities_add.php]: Note: This is where I added my colorPicker stuff

File: interface/usergroup/facility_admin.php [hunk 6]
```diff
@@ -6,240 +6,448 @@
...
+    });
+
+    function displayAlert() {
+      if (document.getElementById('primary_business_entity').checked==false)
+        alert("<?php echo addslashes(xl('Primary Business Entity tax id is used as account id for NewCrop ePrescription. Changing the facility will affect the working in NewCrop.'));?>");
+      else if (document.getElementById('primary_business_entity').checked==true)
+        alert("<?php echo addslashes(xl('Once the Primary Business Facility is set, it should not be changed. Changing the facility will affect the working in NewCrop ePrescription.'));?>");
+    }
+
+    function changeColor(id) {
```
Comment (Jdew192837) [inline interface/usergroup/facility_admin.php]: Note: Where I add my function

**Comment by aethelwulffe** [comment]
Thorough sort of person, aren't you?
Looks good so far...but now we test...
Muahahahahahaha!

**Comment by Jdew192837** [comment]
@aethelwulffe Yep! I think it's worth the extra effort. :)

Did you see my comments on the tab renaming? I'm really lost on that one though...

**Comment by aethelwulffe** [comment]
Haven't been to that issue yet.  There is a lot going on here. 
-this is good work here...but I am afraid you jumped @SoumyadeepJana on the project.  Might wanna give an "oops, sorry".
All the same, good work.

**Comment by SoumyadeepJana** [comment]
@Jdew192837  Good work mate and the pictures and snippets of code changes really help.
@aethelwulffe Its totally fine :) ,Afterall everything is for a common purpose.

**Comment by Jdew192837** [comment]
Sorry about that @SoumyadeepJana! I'll try to be more cognizant next time. :)

File: assets/js/colorpicker2/ColorPicker2.js [hunk 7]
```diff
@@ -1,184 +0,0 @@
-// ===================================================================
```
Comment (teryhill) [inline assets/js/colorpicker2/ColorPicker2.js]: @Jdew192837  is this used anywhere else in the code base? You need to check this before you delete it.

**Comment by teryhill** [comment]
@aethelwulffe let us know how testing goes.

File: assets/js/colorpicker2/ColorPicker2.js (hunk 7 above)
Comment (Jdew192837) [reply assets/js/colorpicker2/ColorPicker2.js]: Nope it was only linked as a script for those two forms. So I just removed it. :) @teryhill

Comment (teryhill) [reply assets/js/colorpicker2/ColorPicker2.js]: Good enough

**Comment by teryhill** [comment]
After @aethelwulffe  gives his review on testing this should be ready for the code base.

**Comment by aethelwulffe** [comment]
@Trodrige 
Merge this after you take a look.  If you can't do merges, then we need to change that.

**Comment by Trodrige** [comment]
@aethelwulffe I can't merge.

**Comment by aethelwulffe** [comment]
@Trodrige You can now.

**Pull request merged**